pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -133,6 +133,9 @@ func (o *LoginOptions) getClientConfig() (*kclient.Config, error) {
 		switch {
 		case o.InsecureTLS:
 			clientConfig.Insecure = true
+			// insecure, clear CA info
```

Comment:
in some cases, I got in situations where both were set... maybe when I had an old CA left from a previous config, so it failed validation, prompted me to be insecure, but still had the old CA data? not sure, but this fixed it